codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AwsServiceIdIntegration.java

```diff
@@ -15,13 +15,17 @@
 
 package software.amazon.smithy.aws.typescript.codegen;
 
+import java.util.Arrays;
```

Comment:
Nit: we alphabetize imports but don't split them based on java.* vs non java.* imports. I thought Checkstyle would have complained about this.

Reply:
Removed the space between imports in [`493c47d` (#1379)](https://github.com/aws/aws-sdk-js-v3/pull/1379/commits/493c47de47997a1fe6175d238453823e56d707db)